theme/src/resources/templates/article_page.hbs

```diff
@@ -9,24 +9,34 @@
 		<div class="row" id="docsArticle">
 			<div class="article-sidenav col-md-3 secondary-font">
 				<div class="sidenav">
-					<div class="sidenav-tree" id="sidenavTree"></div>
-
-					<div class="sidenav-fallback" id="sidenavFallback">
-						{{#with ../section}}
-							<ul class="nav nav-nested">
-								{{#each articles}}
-									<li class="nav-item">
-										<a class="{{#is id ../id}}active semi-bold{{/is}} sidenav-item" href="{{url}}">{{title}}</a>
-									</li>
-								{{/each}}
-							</ul>
-
-							{{#if more_articles}}
-								<div class="nav-item">
-									<a class="semi-bold sidenav-item" href="{{url}}">{{t 'see_more'}}</a>
-								</div>
-							{{/if}}
-						{{/with}}
+					<div class="collapsible-sidebar">
+						<svg aria-label="Hamburger Menu" class="lexicon-icon lexicon-icon-menu-hamburger" role="button" title="Hamburger Menu">
```

Comment:
Instead of `Hamburger Menu` should we just say `Navigation Menu`?

Reply:
Maybe we can say something like "Documentation Navigation Menu" to be more descriptive and differentiate it from the `<nav>` element? That way anyone using a screen reader will know exactly what it's for? Or can you think of a better way to word it?

Reply:
Documentation Navigation Menu sounds fine.  Thanks!

Reply:
1ce51a1, thanks! :bowing_woman: